api/v1alpha1/ai_gateway_route.go

```diff
@@ -227,6 +227,23 @@ type AIGatewayRouteRule struct {
 
 	// Timeouts defines the timeouts that can be configured for an HTTP request.
 	//
+	// AI Gateway provides sensible defaults for AI workloads:
+	// - Request timeout defaults to 60s (instead of Envoy Gateway's default 15s)
+	//   to accommodate longer AI inference times
+	// - If Timeouts is nil, a default request timeout of 60s will be applied
+	// - If Timeouts is specified but Request is nil, the default 60s will be used for Request
+	// - Other timeout fields (BackendRequest) follow Gateway API defaults if not specified
+	//
+	// For streaming responses (like chat completions with stream=true), consider setting
+	// longer timeouts as the response may take time to generate and stream back.
+	//
+	// Example:
+	// ```yaml
```

Comment:
This seems broken on the web
https://deploy-preview-796--envoy-ai-gateway.netlify.app/docs/latest/api/
<img width="1010" alt="image" src="https://github.com/user-attachments/assets/979b27fd-d5a7-4a68-8d22-d7348163e612" />